cmd/frontend/graphqlbackend/users_create.go

```diff
@@ -60,6 +62,14 @@ func (r *createUserResult) ResetPasswordURL(ctx context.Context) (*string, error
 	if !userpasswd.ResetPasswordEnabled() {
 		return nil, nil
 	}
+	var ru string
+
```

Comment:
Make the variable closer to its uses and an empty line to separate logical blocks:

```suggestion

	var ru string
```

Reply:
Good call!